espresso-server/app/build.gradle

```diff
@@ -89,8 +89,10 @@ dependencies {
     testImplementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
 
     androidTestImplementation "androidx.annotation:annotation:$annotation_version"
-    androidTestImplementation "androidx.test.espresso:espresso-contrib:$espresso_version"
-    androidTestImplementation "androidx.test.espresso:espresso-core:$espresso_version"
+    androidTestImplementation("androidx.test.espresso:espresso-contrib:$espresso_version") {
+        transitive = false
```

Comment:
I would say it makes sense to add a comment here or a link the actual PR with a short explanation about why transitive deps are disabled

Reply:
Added in f9b162213e159ac2c510a42d8c47a1b097c4f1c9.